Dockerfile

```
@@ -0,0 +1,40 @@
+    UV_PYTHON=/opt/python3.12/bin/python3.12
+
+ENV UV_CACHE_DIR=/tmp/uv/cache
+# ENV UV_SYSTEM_PYTHON=1
+# ENV UV_PROJECT_ENVIRONMENT=/app 
+
+COPY lib-new lib-new
+RUN uv --directory=lib-new export --format requirements.txt --no-editable --frozen | sed -e 's@^\.@# Elided: .@' /tmp/lib-new-constraints.txt 
+RUN uv --directory=lib-new build -b /tmp/lib-new-constraints.txt -o /tmp/dist --wheel
+
+COPY app-new app-new
+RUN uv --directory=app-new export --format requirements.txt --no-editable --frozen | sed -e 's@^\.@# Elided: .@' /tmp/app-new-constraints.txt 
```

Hadolint (2.12.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)